lotto/client/index.js

```diff
@@ -0,0 +1,67 @@
+import { winningLottoTemplate, randomLottoTemplate } from './template.js'
+
+function Lotto() {
+    const winningLotto = {};
+
+    const getWinningLotto = () => {
+        const round = document.querySelector("#input-lotto-round").value;
+        fetch(`http://localhost:8000/lotto/${round}`)
+            .then(data => data.json())
+            .then(data => {
+                winningLotto.winningNumbers = data.winningNumbers.sort((a, b) => a - b);
+                winningLotto.bonus = data.bonus;
```

Comment:
마찬가지로 수정할 수 있을 것 같네요!
```suggestion
        fetch(`http://localhost:8000/lotto/${round}`)
            .then(response => response.json())
            .then(({winningNumbers, bonus}) => {
                winningLotto.winningNumbers = winningNumbers.sort((a, b) => a - b);
                winningLotto.bonus = bonus;
```

Reply:
변수를 명확히 수정했습니다!